dts/arm/infineon/psoc6/psoc6_02/psoc6_02.dtsi

```diff
@@ -0,0 +1,889 @@
+/*
+ * Copyright (c) 2022 Cypress Semiconductor Corporation (an Infineon company) or
+ * an affiliate of Cypress Semiconductor Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+
+/ {
+	soc {
+		i2s0: i2s@40a10000 {
+			compatible = "infineon,cat1-i2s";
+			reg = <0x40a10000 0x1000>;
+			interrupts = <156 6>;
+			peripheral-id = <0>;
+			status = "disabled";
+		};
+		i2s1: i2s@40a11000 {
+			compatible = "infineon,cat1-i2s";
+			reg = <0x40a11000 0x1000>;
+			interrupts = <158 6>;
+			peripheral-id = <1>;
+			status = "disabled";
+		};
+
+		codec0: codec@40a00000 {
+			compatible = "infineon,cat1-codec";
+			reg = <0x40a00000 0x1000>;
+			interrupts = <157 6>;
+			peripheral-id = <0>;
+			status = "disabled";
+		};
+
+		dmic0: dmic@40a00000 {
+			compatible = "infineon,cat1-dmic";
+			reg = <0x40a00000 0x1000>;
+			interrupts = <157 6>;
+			peripheral-id = <0>;
+			status = "disabled";
+		};
+
+		uid: device_uid@16000600 {
+			compatible = "infineon,cat1-uid";
+			reg = <0x16000600 0xb>;
+			status = "disabled";
+		};
+
+		adc0: adc@409f0000 {
+			compatible = "infineon,cat1-adc";
+			reg = <0x409f0000 0x10000>;
+			interrupts = <155 6>;
+			peripheral-id = <0 0>;
+			status = "disabled";
+		};
+
+		i2c0: i2c@40600000 {
+			compatible = "infineon,cat1-i2c";
+			reg = <0x40600000 0x10000>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			interrupts = <39 6>;
+			peripheral-id = <0>;
+			status = "disabled";
+		};
+		i2c1: i2c@40610000 {
+			compatible = "infineon,cat1-i2c";
```

Comment:
note: where are all the compatibles for the nodes introduced in this commit?

Reply:
another drivers (counter, i2c, pwm) will be added in separate MRs after this one.

Reply:
so all these nodes should be added later, when driver/bindings are added.

Reply:
removed node without drivers support